docs/planning/frontend/src/pages/EvaluatePage.jsx

```
@@ -0,0 +1,250 @@
+            {formData.rating === 1 && "Not for me"}
+            {formData.rating === 2 && "Didn't like it"}
+            {formData.rating === 3 && "It's okay"}
+            {formData.rating === 4 && "Really liked it"}
+            {formData.rating === 5 && "Love it!"}
+          </p>
+        </div>
+
+        {/* Notes */}
+        <div>
+          <label className="label">Notes (optional)</label>
+          <textarea
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)